### ISSUE TITLE
filtered data and distances

### ISSUE THREAD

**Issue by mtwe** [mtwe/fukushima#5]
Hey @awachte and @RyanCaldwell1 ,

So I did a little work on looking at the distances of each safecast point from the 48 different MEXT locations. Here is what I have:

safecast_data_datefiltered.csv
- This is every safecast point that is within the MEXT time frame. I noticed a small error in Ryan's so I fixed it (I think)
- https://mega.nz/#!6JJ3FKxA!IhorMnJqB2C82pz2gxASZ7RY8ty46XMwJ7vsColFgSY

safecast_dists.csv
- For every point in safecast_data_datefiltered.csv, shows the distance to closest MEXT point in km, lat of that MEXT point, long of that MEXT point
- https://mega.nz/#!nVpRwRJI!T-pj0cIUc1ZnR0fDxZnCPm-38mZNDQMZWE5Oi0rJK3A

safecast_data_and_dists.csv
- Basically the combination of the two above. shows the safecast data with three additional fields: distance to closest MEXT point in km, lat of that MEXT point, long of that MEXT point
- https://mega.nz/#!nYQyUKxb!wUZIB2wn-ULGD5ooV0WDJ_MQAXS2rP2rTygYYjLOqaI 

Some initial findings:
- 7044 safecast measurements within 1km of a mext measurement
- 45 safecast measurements within 0.1km of a mext measurement
- This isn't great news but potentially something we can work with. We'll have to try and match up time and space next
